jack.py

```diff
@@ -1533,6 +1562,65 @@ def get_array(self):
         return np.frombuffer(self.get_buffer(), dtype=np.float32)
 
 
+def _maybe_bytes(data):
+    if type(data) is bytes:
+        data_ptr = _ffi.new("unsigned char[]", len(data))
+        data_ptr[0:len(data)] = data
+        return data_ptr
+    return data
+
+def _returns_bytes(fun):
+    def wrapper(self, cnt):
+        ret_cnt, data = fun(self, cnt)
+        return bytes(_ffi.buffer(data, ret_cnt))
+    return wrapper
+
+class RingBuffer(object):
+    def __init__(self, size=256):
```

Comment:
No strong reason, just wanted a quick way to define a buffer without specifying the size. Removing it.